lib/tasks/data_migrations.rake

```diff
@@ -1,70 +1,4 @@
 namespace :data do
-  namespace :migrate_info_hash do
```

Comment:
Please move to a separate PR

Reply:
Updated in https://github.com/bikeindex/bike_index/pull/1296